durable/durable-driver.sh

```diff
@@ -26,11 +26,13 @@ AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID-.}
 AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY-.}
 AWS_REGION=${AWS_REGION-us-east-1}
 
-docker run \
+docker run --network="host" \
```

Comment:
We should add a TODO or at least a comment explaining we are using the host network to make portforwarding work. And that we can remove that once we use public URLs in the future